src/scripts/advanced_overlay.user.js

```diff
@@ -11,8 +11,12 @@
 // @grant        none
 // ==/UserScript==
 
-let width = "2500px";
```

Comment:
Moved To
https://github.com/PlaceDE-Official/place-overlay/pull/22/files#diff-af4920c0010114b300e83f3e59dd1eba073699a34397f27a0170570ea6ebd0a6R61-R62